02/styles/global.css

```diff
@@ -14,13 +25,26 @@
 }
 
 .a {
-    border-color: #ffffba
+    border-color: #ffffba;
+    grid-area: A;
 }
 
 .b {
     border-color: #ffdfba;
+    grid-area: B;
 }
 
 .c {
     border-color: #baffc9;
+    grid-area: C;
+}
+
+@media (min-width:600px) {
+    body {
+        grid-template-columns: minmax(auto, 1200px) minmax(auto, 1200px);
```

Comment:
Bardziej chodziło o to, że zawartość nie ma być większa niż 1200px ( w sumie), a nie każda z kolumn ;)